ext/fs/ops.rs

```diff
@@ -455,6 +455,23 @@ where
   Ok(SerializableStat::from(stat))
 }
 
+#[op2(fast)]
+pub fn op_fs_exists_sync<P>(
+  state: &mut OpState,
+  #[string] path: String,
+) -> Result<bool, AnyError>
+where
+  P: FsPermissions + 'static,
+{
+  let path = PathBuf::from(path);
+  state
+    .borrow_mut::<P>()
+    // currently only used by node:fs
+    .check_read(&path, "node:fs.existsSync()")?;
```

Comment:
Should I move this to ext/node?

Reply:
Yes, I think we should move it to `ext/node` for the time being.